src/components/form/HdCheckboxCard.vue

```diff
@@ -0,0 +1,315 @@
+<template>
+  <label
+    ref="label"
+    :for="inputName"
+    @click="$refs.checkbox.focus()"
+    @keydown.prevent.enter="$refs.label.click()"
```

Comment:
I don't think that we should be listening to a `enter` key. On a [regular checkbox](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox) the enter key does not work, only the `space` key... Usually we should use the checkboxes inside a form, and, in general, the enter should trigger the form submit. What do you think?

Reply:
Thanks a lot, man, you are totally right =D
4fdfdd6